python/pyspark/sql/tests/connect/test_connect_column.py

```diff
@@ -202,6 +203,9 @@ def test_literal_null(self):
             self.assertTrue(lit_null._value is None)
             self.assertEqual(dataType, lit_null._dataType)
 
+            cdf = self.connect.range(0, 1).select(Column(lit_null))
```

Comment:
for each datatype supported in `LiteralExpression`, create a df containing the literal column and check the schema